dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -8,7 +8,6 @@ import {
 } from '@guardian/source-foundations';
 import { Button, SvgShareCallout } from '@guardian/source-react-components';
 import { useState } from 'react';
-import { sanitiseHTML } from '../../../../src/model/sanitise';
```

Comment:
@sophie-macmillan had to remove sanitise html here as it is not supported in react components in DCR. The norm seems to be just simply use `dangerouslySetInnerHtml`